tests/functional/kmip/serverside_encryption.js

```diff
@@ -6,21 +6,28 @@ const http = require('http');
 const https = require('https');
 const assert = require('assert');
 const logger = { info: msg => process.stdout.write(`${msg}\n`) };
+const async = require('async');
```

Comment:
You might want to add test failing if 'x-amz-scal-server-side-encryption' is not equal to 'AES256'.

Reply:
Allowed values for `x-amz-scal-server-side-encryption` are `AES256` and `aws:kms`, we do not allow object level encryption and the change/improvement is to allow the header in object level requests and it has to be `AES256`. I have added unit tests for the utility and it covers this case.